src/services/getBatteryIcon.js

```diff
@@ -1,13 +1,12 @@
-import React from 'react';
+import React, { lazy } from 'react';
 import PT from 'prop-types';
 
-import MainBatteryEmptySvg from 'app/static/vectors/batteries/main-battery-empty.svg';
-import MainBatteryHalfSvg from 'app/static/vectors/batteries/main-battery-half.svg';
-import MainBatteryFullSvg from 'app/static/vectors/batteries/main-battery-full.svg';
-import LockBatteryEmptySvg from 'app/static/vectors/batteries/lock-battery-empty.svg';
-import LockBatteryHalfSvg from 'app/static/vectors/batteries/lock-battery-half.svg';
-import LockBatteryFullSvg from 'app/static/vectors/batteries/lock-battery-full.svg';
-
+const MainBatteryEmptySvg = lazy(() => import('app/static/vectors/batteries/main-battery-empty.svg'));
+const MainBatteryHalfSvg = lazy(() => import('app/static/vectors/batteries/main-battery-half.svg'));
+const MainBatteryFullSvg = lazy(() => import('app/static/vectors/batteries/main-battery-full.svg'));
+const LockBatteryEmptySvg = lazy(() => import('app/static/vectors/batteries/lock-battery-empty.svg'));
+const LockBatteryHalfSvg = lazy(() => import('app/static/vectors/batteries/lock-battery-half.svg'));
+const LockBatteryFullSvg = lazy(() => import('app/static/vectors/batteries/lock-battery-full.svg'));
```

Comment:
I would not recommend "splitting" these icons from your bundle, as they are very small chunks of codes, they are still in code context, you won't get the images exported like `png`s or `jpg`s, and every lazy import here will be a separate `js` file, which is going to result in lower performance. I think it's not necessary, webpack does a great job here. Maybe you can modify the webpack config to export `svg`s just like `png`s instead.